warehouse/admin/templates/admin/helpscout/app.html

```diff
@@ -11,6 +11,7 @@
  # See the License for the specific language governing permissions and
  # limitations under the License.
 -#}
+{%- for user in users -%}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏I recall some setting we could use to auto-set the `{%-` tag strip behaviors for all templates, we might wanna look that up at some point.

Reply:
just did this to make the existing test pass 😂, its HTML in the end so I'm not sure how important setting such a default makes.